tools/eslint-multi-format/eslint.config.js

```diff
@@ -0,0 +1,26 @@
+import tseslint from 'typescript-eslint';
+import baseConfig from '../../eslint.config.js';
+
+export default tseslint.config(
+  ...baseConfig,
+  {
+    files: ['**/*.ts'],
+    languageOptions: {
+      parserOptions: {
+        projectService: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+  {
+    files: ['**/*.json'],
+    rules: {
+      '@nx/dependency-checks': [
+        'error',
+        {
+          ignoredDependencies: ['@nx/vite', 'vite', 'vite-plugin-dts'],
+        },
+      ],
```

Comment:
These exceptions shouldn't be needed.

```suggestion
      '@nx/dependency-checks': 'error',
```

The problem is that `vite.config.ts` is incorrectly identified as a production dependency. You can fix that by adding `"!{projectRoot}/vite.config.ts"` to `production` input patterns in `nx.json`.

https://github.com/code-pushup/cli/blob/158ee7a41b1b00db4271ac26847187e7eb5cd1fa/nx.json#L100-L114

However, it may be best to not introduce new config file patterns - see previous comment.

Reply:
Did

Reply:
Thanks for pointing it out!